### ISSUE TITLE
Make all assets and strings configurable

### ISSUE THREAD

**Issue by jyksnw** [OakRidgeFMC/frontend#13]
The eventual goal of both the frontend and backend is that they can be deployed for use by any church at virtually no cost. Because of this all site-specific strings (title's, alt text, logos, etc..) should be made easily configurable either through the use of a configuration file or through a payload retrieved from the backend.

**Comment by cameronapak** [comment]
I have seen Hugo do something like this. They have a .toml file that they store hardcoded strings and other configurations.

**Comment by jyksnw** [comment]
Since this site is generated on the client based on the API we could achieve this by setting up a settings API endpoint that gathers common strings or we can bundle something like a config.json with the site that is queried at load time. We can further discuss if this should be bundled into the site itself or should be fetched from the API I would need to think through the pro's and con's of each.

I haven't added an issue for this yet, but one of the future enhancements is to set up a service worker to cache JSON data, API responses, and assets locally so that the site appears to be fully working even if the client doesn't have internet. This also aids in the goal of cutting down on cost (fewer API calls) and increasing performance of the site.
